src/styles/tailwindcss/components/from-group.css

```
@@ -0,0 +1,20 @@
+@layer components {
+  .tw-from-group {
```

there is a typo in file and class names

```suggestion
  .tw-form-group {
```